bootstrap/100_final_perms.sql

```diff
@@ -13,3 +13,9 @@ grant select on all views in schema CATALOG to APPLICATION ROLE READ_ONLY;
 grant usage on all functions in schema TOOLS to APPLICATION ROLE READ_ONLY;
 grant usage on all functions in schema TOOLS to APPLICATION ROLE ADMIN;
 grant usage on all functions in schema TOOLS to APPLICATION ROLE PUBLIC_;
+
+-- perms for service account
+-- TODO should we limit to individual procs,views?
```

Comment:
Aside from the setup-related procs, I can't think of a reason that we'd want to limit access for the service role.

Reply:
Only the principle of least privilege. Perhaps we only need to be able to modify the crud and settings, not make changes to other admin procs. Additionally, perhaps they don't want to give us access to all data, just the report card related stuff.

Either way, I want to defer to a later PR, when we know more